src/apps/core/admin.py

```diff
@@ -78,16 +126,25 @@ class DatasetAdmin(AbstractDatasetPropertyBaseAdmin, SimpleHistoryAdmin):
         "modified",
     )
     list_filter = (
-        "language",
-        "keyword",
         "created",
         "modified",
         "issued",
         "deprecated",
         "state",
+        "data_catalog",
+    )
+    autocomplete_fields = ['language', "theme", "field_of_science"]
+    readonly_fields = (
+        "dataset_versions",
+        "other_identifiers",
+        "last_modified_by",
+        "access_rights",
+        "draft_of",
     )
-    filter_horizontal = ("language", "theme", "field_of_science")
     list_select_related = ("access_rights", "data_catalog", "metadata_owner")
+    search_fields = ["title__values", "keyword"]
+
+    # inlines = [LanguageInline]
```

Comment:
uncomment the line
```suggestion
    inlines = [LanguageInline]
```

Reply:
I think this was an artifact from earlier when I tried to make admin more usable with Inlines. Adding the following class:

```python
class LanguageInline(admin.TabularInline):
    model = Dataset.language.through
```
And uncommenting the line will add new inline on the end of the Dataset Admin view:
![Screenshot from 2024-08-30 10-32-35](https://github.com/user-attachments/assets/8bbc0433-4cfb-488c-9295-677d3e65c411)
But there is already field for the Language too:
![Screenshot from 2024-08-30 10-33-02](https://github.com/user-attachments/assets/f3e432e2-dc18-40c8-bc64-5c6377228d6e)

So should I just delete the commented line or add the Inline Class and uncomment the Inline in DatasetAdmin class?

Reply:
multiselect looks nicer, so delete the commented line.

Reply:
Done in latest commit